backend/src/oe/api/routes.ts

```diff
@@ -245,6 +246,20 @@ class OpEnergyRoutes {
     logger.info( `${UUID}: PROFILE: end: $getBlockByHash`);
   }
 
+  private async $getBlockByHeight(req: Request, res: Response) {
+    const UUID = await opEnergyApiService.$generateRandomHash();
+    try {
+      logger.info( `${UUID}: PROFILE: start: $getBlockByHeight`);
+      const { height } = req.params;
```

Comment:
what if height is not present in the params? please add proper error handling

Reply:
then height is undefined. Added check for parseInt(height) being NaN